### ISSUE TITLE
Package ramsey/array_column is abandoned

### ISSUE THREAD

**Issue by netbrothers-tr** [php-edifact/edifact#110]
Composer gives this message upon installation:
```
Package ramsey/array_column is abandoned, you should avoid using it. Use it-for-free/array_column instead.
```
On the [package site](https://packagist.org/packages/ramsey/array_column) it also states that it's abandoned and one should use [it-for-free/array_column](https://packagist.org/packages/it-for-free/array_column) instead. I don't think this is very urgent, but should be addressed.

**Comment by sabas** [comment]
@netbrothers-tr uhm are you sure it is required by the edifact package? It's not an explicitly required package so I wonder where to look... Thanks for any hint you may give me...

**Comment by netbrothers-tr** [comment]
@sabas, I checked again and found out the problem doesn't occur with the latest version (0.5.1). We had a fixed requirement of  `php-edifact/edifact:0.4.1` and that required an older version of `voku/arrayy` which in turn required `ramsey/array_column`. But [this is all history now](https://github.com/voku/Arrayy/commit/da8399f0694f66e73644a7f37dc1f60fa70da231#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34) and I will close the issue. Thank you for publishing and maintaining this library!
